### DISCUSSION TITLE
TGN for node classification

### DISCUSSION THREAD

**Discussion by lisiq** [pyg-team/pytorch_geometric#4761] [Q&A]
Hi,

Thank you very much for the detailed examples! They are very helpful. I saw that there is the link prediction task using TGN [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/tgn.py). Will there be a node classification example provided in the future? 
Thank you in advance!

Best regards,
Lisi

**Comment by Padarn** [comment]
Hey @lisiq, I'm guessing there is no concrete plan for that. But I encourage you to add one!
